UPDATE: tinted-theming gallery links in various README.md

dtgagnon opened the pull request:
Base16 gallery link was broken. Updated to align with working base24 gallery domain, for base16 gallery page.

dtgagnon committed:
Update README.md

Base16 gallery link was broken. Updated to align with working base24 gallery domain, for base16 gallery page.

dtgagnon committed:
Update README.md

Updated gallery link

JamyGolden commented on base16/README.md:
```diff
@@ -1,7 +1,7 @@
 
 Have a look at the
-[Gallery](https://tinted-theming.github.io/base16-gallery/) to preview these colorschemes.
+[Gallery](https://nico-i.github.io/scheme-viewer/base16/) to preview these colorschemes.
```
```suggestion
[Gallery](https://tinted-theming.github.io/tinted-gallery/) to preview these colorschemes.
```

JamyGolden commented on README.md:
```diff
@@ -2,7 +2,7 @@
 Scheme families:
 
-- [base16] ([gallery](https://tinted-theming.github.io/base16-gallery/))
+- [base16] ([gallery](https://nico-i.github.io/scheme-viewer/base16/))
```
```suggestion
- [base16] ([gallery](https://tinted-theming.github.io/tinted-gallery/))
```

JamyGolden reviewed (changes requested):
I added dark mode support and base24 support to the gallery and changed the name to tinted-gallery but forgot to update in some of the readmes, I'll do that now.

Now that base24 is supported in the gallery (https://tinted-theming.github.io/tinted-gallery/ - group by -> base24), I feel like we can start looking at replacing the scheme-viewer links with tinted-gallery since it supports both scheme systems now.

dtgagnon committed:
FIX: Links to tinted-gallery

Co-authored-by: Jamy <jamy@jamygolden.com>

dtgagnon committed:
consolidating base16 + base24 README.md gallery links

https://tinted-theming.github.io/tinted-gallery/` supports viewing both base16 and base24 galleries.
